app/views/application/_notebook_run.slim

```diff
@@ -1,5 +1,5 @@
 -if @user.member?
   div.ribbon-wrapper
     div.github-fork-ribbon
-      a.run_jupyter href='/notebook/#{ nb.uuid }' Run in Jupyter
-      input type='hidden' value="#{(defined?(ref) && !ref.blank? ? ('?ref=' + ref) : '')}"
+      a.run_jupyter href="#{uuid_notebook_path(nb.uuid)}" Run in Jupyter
```

Comment:
This causes things to break because it has a trailing /uuid on the URL so the code in run_in_jupyter.erb fails when it tries to pull the actual UUID out of the URL which then causes the notebook to not be found.
Old format of this link: https://<domain>/notebook/<uuid>
This code produces: https://<domain>/notebook/<uuid>/uuid
The code that it breaks is:
https://github.com/nbgallery/nbgallery/blob/38c8c48ed2040ee120e16be33011242273c717a4/app/assets/javascripts/run_in_jupyter.js.erb#L17